### PR TITLE
3.14.2

### PR DESCRIPTION
**Pull request by fy1900**
Added one new icon. Can you please help review the changes?

### CODE REVIEW

File: docs/keywords.json [hunk 1]
```diff
@@ -7278,15 +7278,24 @@
...
+      "group form plus",
+      "add group",
       "create group"
     ],
     "category":"Forms",
     "release":"3.14.1"
+  },
+  "unselected":{
+    "alias":[
+      "unselection",
```
Comment (johnmgriffith) [inline docs/keywords.json:7294]: Add "3.14.2" and "generic" here please

**Review by johnmgriffith — CHANGES_REQUESTED** [review]
Just add the new tags, and it should be good!

**Review by johnmgriffith — APPROVED** [review]
Perfect! G2G 🎉

**Comment by fy1900** [comment]
**Added icon**
unselected

Screen Shot 2020-12-08 at 12 54 45 PM

**Pull request merged**